### DISCUSSION TITLE
Computed text, inside markdown: is there an example in the documentation?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#865] [Q&A] [answered]
Hi to all,
probably this is a stupid question.

Is there a way, for an HTML output, to print the value of a variable inside a markdown text.

Something like the below example.

~~~
```{python}
age = 49;
```

My name is **Andrea** and I'm {age} years old

~~~

Thank you

**Comment by jjallaire** [comment, marked as answer]
Yes, but its current a bit awkward (docs here: https://quarto.org/docs/computations/execution-options.html#jupyter)

For your example it would look like this:

````
```{python}
#| echo: false
age = 49
from IPython.display import display, Markdown
display(Markdown("""
My name is **Andrea** and I'm {age} years old
""".format(age = age)))
```


````

**Reply by aborruso** [reply]
Thank you.

The way  observable does it, it's really great. I will use OJS inside quarto ;)

![image](https://user-images.githubusercontent.com/30607/168303166-0d4b3837-2e33-4754-a1ed-55456ec0e9dc.png)

**Reply by mcanouil** [reply]
@rabenojha Please do not ping people directly. Also if you question is different or the answer does not match what you want to do, do open a new discussion and provide concrete small examples and code if appropriate. Thank you!
Note that sharing screenshots instead of code is unusable, in addition to not being good for accessibility.
